From 169f1f6002ea848e5a1b18f7c205446b0be9af53 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Tue, 2 Mar 2021 14:50:40 +0100
Subject: [PATCH 2/2] Revert "disable BUILD_TESTING"

This reverts commit 08d1e5fbca3fa3a740eac384092c02f40821544d.

doesn't work well enough: the issue still persist in
ws/devel_isolated/sot-core/lib/cmake/sot-core/sot-coreConfig.cmake

will add a patch instead.
---
 debian/rules.em | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 8f9ef68..3342408 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -30,7 +30,6 @@ override_dh_auto_configure:
 	# CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
-		-DBUILD_TESTING=OFF \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
 
-- 
2.31.1

